Source/Controllers/MainViewControllers/ConnectionView/SPConnectionController.m

```diff
@@ -1519,21 +1512,6 @@ - (void)_saveCurrentDetailsCreatingNewFavorite:(BOOL)createNewFavorite validateD
 	[[NSNotificationCenter defaultCenter] postNotificationName:SPConnectionFavoritesChangedNotification object:self];
 }
 
-/**
- * Check the host field and ensure it isn't set to 'localhost' for non-socket connections.
- */
-- (BOOL)_checkHost {
-	if ([self type] != SPSSHTunnelConnection && [self type] != SPSocketConnection && [[self host] isEqualToString:@"localhost"]) {
-		[NSAlert createAlertWithTitle:NSLocalizedString(@"You have entered 'localhost' for a non-socket connection", @"title of error when using 'localhost' for a network connection") message:NSLocalizedString(@"To MySQL, 'localhost' is a special host and means that a socket connection should be used.\n\nDid you mean to use a socket connection, or to connect to the local machine via a port?  If you meant to connect via a port, '127.0.0.1' should be used instead of 'localhost'.", @"message of error when using 'localhost' for a network connection") primaryButtonTitle:NSLocalizedString(@"Use 127.0.0.1", @"Use 127.0.0.1 button") secondaryButtonTitle:NSLocalizedString(@"Connect via socket", @"Connect via socket button") primaryButtonHandler:^{
```

Comment:
I kind of liked this modal when I entered `localhost` and it changed that for me to 127.0.0.1 for TCP/IP 😄

Reply:
Lol well now localhost should just work so at least maybe it doesn't take need to be replaced in the first place? 😆
Could add the modal back, but it got annoying and kept popping up when it shouldn't

Reply:
Yeah in my setup it does work with both options. :)